packages/design-system/src/components/tooltip/index.js

```diff
@@ -133,6 +134,17 @@ function Tooltip({
 
     setArrowDelta(delta);
   }, []);
+  // When near the bottom of the viewport and the tooltip is placed on the bottom we want to force the tooltip to the top as to not
+  // cutoff the contents of the tooltip.
+  const positionPlacement = useCallback(() => {
+    const tooltipElBoundingBox = tooltipRef.current?.getBoundingClientRect();
+    if (
+      placement.startsWith('bottom') &&
+      tooltipElBoundingBox?.bottom > window.visualViewport.height
```

Comment:
out of curiosity, why'd you use `window.visualViewport` here over `window.innerHeight` ? just never seen `visualViewport` before is why I ask

Reply:
Honestly, I don't have a "good" reason. And after looking at the docs, `innerHeight` is probably best since explorer doesn't support `visualViewport`

Reply:
I did find this info though. https://developers.google.com/web/updates/2017/09/visual-viewport-api

Reply:
We don't support IE, FWIW

Reply:
Oh, jsdom doesn't mock `visualViewport`, so JS unit tests are currently failing because of that. Would need to use some optional chaining / fallback for that. Or just use `innerWidth`, which jsdom provides.